packages/api/src/schema/schema.yaml

```diff
@@ -811,6 +815,381 @@ components:
+                example: 23.8238
+              bitrate:
+                type: number
+                description: Bitrate of the track in bits per second
+                example: 1000000
+              channels:
+                type: number
+                description: Amount of audio channels in the track
+                example: 2
+              sampleRate:
+                type: number
+                description:
+                  Sample rate of the track in samples per second - only for
+                  audio tracks
+                example: 44100
+              bitDepth:
+                type: number
+                description: Bit depth of the track - only for audio tracks
+                example: 16
+              codec:
+                type: string
+                description: Codec of the track
+                example: aac
+        profiles:
+          type: object
+          additionalProperties: false
+          description: HLS Profile manifests associated with the asset
+          properties:
+            kind:
+              type: string
+              description: Type of the profile
+              example: manifest
+            name:
+              type: string
+              description: Filename of the manifest
+              example: variant_1080.m3u8
+            width:
+              type: number
+              description: width of the profile
+              example: 1920
+            height:
+              type: number
+              description: height of the profile
+              example: 1080
+            fps:
+              type: number
+              description: Frame rate of the profile
+              example: 30
+            id:
+              type: string
+              description: ID of the profile
+              example: ABC123
+            duration:
+              type: number
+              description: Duration of the profile in seconds
+              example: 23.8238
+            bitrate:
+              type: number
+              description: Bitrate of the profile in bits per second
+              example: 1000000
+        playbackUrl:
+          type: string
+          description: Playback url of the playlist manifest
+          example: https://cdn.livepeer.com/ABC123/playlist.m3u8
+        status:
+          type: string
+          description: Status of the asset
+          example: ready
+        originTaskId:
+          type: string
+          description: Task ID of the task that created the asset
+          example: 09F8B46C-61A0-4254-9875-F71F4C605BC7
+        relatedTasks:
+          type: array
+          description: List of task IDs associated with the asset
+          items:
+            $ref: "#/components/schemas/task"
+        sourceAssetId:
+          type: string
+          description:
+            ID of the source asset (root) - If missing, this is a root asset
+          example: 09F8B46C-61A0-4254-9875-F71F4C605BC7
+        nft:
+          type: array
```

Comment:
I think an nft should be just another immutable asset pointing to the file on a decentralized storage, with whatever extra nft parameters saved on an equivalent "nft spec" field (which could be `nftSpec` or just `nft` as I'm realizing now).

If there are multiple NFTs minted from the same asset, there would be multiple separate assets created, each with their own NFT spec (but possibly the same storage?)

Reply:
I agree that NFTs should be another asset, let's remove it and implement it later